docs/Pitfalls.asciidoc

```diff
@@ -53,3 +53,45 @@ impacted by snapshot creation. Loading VM snapshots can also have an impact on
 SUT behavior as the execution of the first step after loading a snapshot might
 be delayed. This can lead to problems if the executed tests do not foresee an
 appropriate timeout margin.
+
+== DB migration from sqlite to postgreSQL
+In order to start using postgreSQL DBMS with openQA one need to perform following steps:
```

Comment:
"DBMS" is a technical term/abbreviation which is not explained. Better explain that.

Reply:
Not really, the term can also be removed. Explaining what a DBMS is out of the scope of this.